INSTALL.md

```diff
@@ -1,8 +1,22 @@
 # Install Open Source Event Manager
-All the information that you need to install OSEM. If you have any problems with installing don't hesitate to [contact us](https://github.com/openSUSE/osem#contact)
+This file contains all the information you need to install OSEM. If you run into any problems, do not hesitate to [contact us](https://github.com/openSUSE/osem#contact).
+
+If you encounter
+```
+An error occurred while installing yajl-ruby (1.2.0), and Bundler cannot continue.
+Make sure that `gem install yajl-ruby -v '1.2.0'` succeeds before bundling.
```

Comment:
This is the standard error that is shown when `bundle install` fails, right? And it could happen with any other gem too.